Neuro/Minigames/MinigamesHandler.cs

```diff
@@ -35,6 +35,37 @@ public IEnumerator CompleteMinigame(PlayerTask task, Minigame minigame)
         NeuroPlugin.Instance.Tasks.UpdatePathToTask(task);
     }
 
+    public IEnumerator CompleteDoorMinigame(Minigame minigame)
+    {
+        if (minigame.TryCast<DoorBreakerGame>() is { } breakerGame)
+        {
+            yield return new WaitForSeconds(Random.RandomRange(1f, 2.5f));
```

Comment:
Shouldn't we add `DoorBreakerGame` to `GetTimeToComplete`? It'd be nice if there's only one place to update wait times.

Reply:
I totally agree but the issue is that it is not part of the `TaskType` enum, which we don't own. I *could* reuse `TaskType.ResetBreakers` but it doesn't take 10-15 seconds to do the local breaker doors, while the RPC electrical breakers can take up to 10-15 seconds.

If its possible to postfix an enum then I would be happy to do it, but as far as I can tell that's not possible :(